### PR TITLE
docs(instances): Fix Ubuntu Noble not reachable after reboot (int-add-instances)

### PR DESCRIPTION
**Pull request by karibou**
### Your checklist for this pull request

- [ x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Add a troubleshooting document to explain how to fix unreachable Ubuntu Noble instances.

Please describe what you added or changed.
Added a new document

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+meta:
+  title: Fix unreachable Ubuntu Noble Instances following a reboot
+  description: This page helps recover or avoid unreachable Ubuntu 24.04 Noble Instances after a reboot
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:4]:
```suggestion
  description: This page helps you to recover or avoid unreachable Ubuntu 24.04 Noble Instances after a reboot
```

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+content:
+  h1: Fix unreachable Ubuntu Noble Instances following a reboot
+  paragraph: This page helps recover or avoid unreachable Ubuntu 24.04 Noble Instances after a reboot
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:7]:
```suggestion
  paragraph: This page helps you to recover or avoid unreachable Ubuntu 24.04 Noble Instances after a reboot
```

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+tags: ubuntu-noble noble 24.04 reboot
+dates:
+  validation: tbd
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:10]: need date

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  posted: tbd
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:11]: Need date

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  - compute
+---
+
+The newest Ubuntu 24.04 Noble Numbat Instances may become unreachable after a reboot. This distribution now uses dhcpcd which may encounter longer delays to reapply its DHCP lease after a reboot.<br />
+
+Instances created after June 11th 2024 will not encounter this problem.
+
+## Your Instance has not been rebooted
+
+You can avoid this situation by increasing the dhcpcd timeout it its configuration file
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:22]:
```suggestion
You can avoid this situation by increasing the dhcpcd timeout in its configuration file
```

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+# OR generate Stable Private IPv6 Addresses based from the DUID
+slaac private
+timeout 10
+```
+
+## Your Instance has rebooted and is no longer reachable
+
+[Rescue mode](/compute/instances/concepts/#rescue-mode) allows you to boot your Instance into a temporary operating system, running from the Instance's RAM. Rescue mode is often used to diagnose and resolve issues such as faulty kernel configurations, networking issues, firewall configurations, and more.
+
+1. Switch the Instance's `boot-type` to `rescue` and reboot your Instance into rescue mode using the CLI-Tools:
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:43]:
```suggestion
1. Switch the Instance's `boot-type` to `rescue` and reboot your Instance into rescue mode using the [CLI](https://www.scaleway.com/en/docs/developer-tools/scaleway-cli/quickstart/):
```

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    8 0 9765625 sda
+    8 1 9634536 sda1
+    8 14 3072 sda14
+    8 15 126976 sda15
+    ```
+    Then mount the partitions:
+    ```
+    mount /dev/sda1 /mnt
+    ```
+3. Once mounted, use the `chroot` command to get into your Instances' root filesystem. You can then change the timeout value for dhcpcd as shown above :
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:66]:
```suggestion
3. Once mounted, use the `chroot` command to get into your Instance's root filesystem. You can then change the timeout value for dhcpcd as shown above :
```

**Review by RoRoJ — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 8]
```diff
@@ -0,0 +1,93 @@
+---
+meta:
+  title: Fix unreachable Ubuntu Noble Instances following a reboot
+  description: This page helps you to recover or avoid unreachable Ubuntu 24.04 Noble Instances after a reboot
+content:
+  h1: Fix unreachable Ubuntu Noble Instances following a reboot
+  paragraph: This page helps you to recover or avoid unreachable Ubuntu 24.04 Noble Instances after a reboot
+tags: ubuntu-noble noble 24.04 reboot
+dates:
+  validation: tbd
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:10]:
```suggestion
  validation: 2024-06-11
```

File: compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+  posted: tbd
```
Comment (RoRoJ) [inline compute/instances/troubleshooting/fix-unreachable-noble-after-reboot.mdx:11]:
```suggestion
  posted: 2024-06-11 
```

**Pull request merged**